.github/workflows/ci.yml

```diff
@@ -35,12 +35,15 @@ jobs:
       run: poetry run python setup.py py2app --emulate-shell-environment
 
     - name: Compress app
-      run: zip -r timelapse.zip dist/Timelapse.app
+      shell: bash
+      run: |
+        ( cd dist; zip -r timelapse.zip Timelapse.app )
```

Comment:
`bash` should be the default, no? Also I'm not sure why we need the parentheses. Isn't this equivalent?

```
    - name: Compress app
      run: |
        cd dist
        zip -r timelapse.zip Timelapse.app
        mv dist/timelapse.zip ./
```

Reply:
I didn't know which shell was default and I wasn't sure if the shell environment was preserved across different lines in run (like a typical shell session), so this is just very conservative code. I'll change it to your suggestion just now.